libs/hmpb/src/generate_fixtures.ts

```diff
@@ -1,25 +1,23 @@
-import { mkdir, rm, writeFile } from 'node:fs/promises';
 import { iter } from '@votingworks/basics';
 import { writeImageData } from '@votingworks/image-utils';
-import {
-  HmpbBallotPaperSize,
-  HmpbBallotPaperSizeSchema,
-  unsafeParse,
-} from '@votingworks/types';
+import { HmpbBallotPaperSize } from '@votingworks/types';
+import { mkdir, rm, writeFile } from 'node:fs/promises';
 import { dirname } from 'node:path';
 import {
   AllBubbleBallotFixtures,
   allBubbleBallotFixtures,
 } from './all_bubble_ballot_fixtures';
 import {
+  nhGeneralElectionFixtures,
   timingMarkPaperFixtures,
   vxFamousNamesFixtures,
   vxGeneralElectionFixtures,
-  nhGeneralElectionFixtures,
   vxPrimaryElectionFixtures,
 } from './ballot_fixtures';
-import { Renderer } from './renderer';
+import { convertPdfToCmyk } from './pdf_conversion';
```

Comment:
This function doesn't appear to be used in this file. I would have expected the linter to catch this

Reply:
[The linter did catch it.](https://app.circleci.com/pipelines/github/votingworks/vxsuite/21174/workflows/dc5e5b69-b14d-4a0c-8bd4-7f5a5371a772/jobs/922357) I've been fiddling with my dev setup, so I haven't been using the pre-commit hooks.